### ISSUE TITLE
Python Gradio app doesn't show in viewer the first time it's run

### ISSUE THREAD

**Issue by jonvanausdeln** [posit-dev/positron#4952]
#### Positron and OS details:

Windows 11
Positron [2024.10.0-13](https://github.com/posit-dev/positron/releases/tag/2024.10.0-13)

#### Interpreter details:

Python 3.12.2

## Describe the issue:

Running a simple gradio application doesn't open up in viewer pane. (similar to #4890)
But it does if you run it a second time and every time after that.

## Steps to reproduce the issue:

```
import gradio as gr
def image_classifier(inp):
    return {'cat': 0.3, 'dog': 0.7}
demo = gr.Interface(fn=image_classifier, inputs="image", outputs="label")
demo.launch()
```

_note_ - in the video below, I had to delete the `Gradio` terminal, and then click play again to get it to work.  Sometimes just hitting play a second time without deleting the terminal gets it to work

https://github.com/user-attachments/assets/e1bc4136-2235-4d68-8653-875197a5aebd

## Expected or desired behavior:

App to open in viewer after hitting the run button. If I click "follow link" from the terminal, I can open the app in the viewer, browser, etc.

## Were there any error messages in the UI, Output panel, or Developer Tools console?
This is the console log from when just clicking the run button a second time gets it to display in viewer.
[vscode-app-1728427470358.log](https://github.com/user-attachments/files/17299935/vscode-app-1728427470358.log)

**Comment by sharon-wang** [comment]
This might have been fixed in #4978 as I increased the timeout for python app startup. I've been able to run this app on Mac Desktop and in Positron on Workbench on Linux.

**Comment by jonvanausdeln** [comment]
### Verified Fixed
Positron Version(s) : [2024.11.0-140](https://github.com/posit-dev/positron/releases/tag/2024.11.0-140)
OS Version(s) : Windows 11

### Test scenario(s) 

Gradio app now works as expected the first time running the app.



### Link(s) to TestRail test cases run or created:
